### PR TITLE
[Package add] vega-strike-git & vega-strike-engine-git

### PR DESCRIPTION
**Pull request by echometerain**

### CODE REVIEW

**Review by Henryws — APPROVED** [review]

**Comment by wizard-28** [comment]
**DO NOT MERGE TILL #537 IS MERGED**

**Review by wizard-28 — CHANGES_REQUESTED** [review]
Follow the instructions to setup your pacscript development environment listed [here](https://github.com/pacstall/pacstall-programs#how-to-setup-the-environment-for-pacscript-development)

After doing so try staging (`git add`) this file (make a small dummy modification) then try to commit it (after running `pre-commit install`). When the pre-commit hooks fail (if they fail), fix the errors.

File: packages/vega-strike-engine-git/vega-strike-engine-git.pacscript [hunk 1]
```diff
@@ -8,26 +8,26 @@ description="Vega Strike engine"
...
 
 build() {
-	mkdir build
-	cd build
-	cmake "../engine"
-	cmake --build "./build" -j$(nproc) -DCMAKE_BUILD_TYPE=Release	
-	make
-	cd ..
+  mkdir build
+  cd build || exit
```
Comment (wizard-28) [inline packages/vega-strike-engine-git/vega-strike-engine-git.pacscript:21]:
```suggestion
  cd build || return 1
```

File: packages/vega-strike-git/vega-strike-git.pacscript [hunk 2]
```diff
@@ -6,18 +6,18 @@ pacdeps="vega-strike-engine-git"
 maintainer="echometerain <echometer@disroot.org>"
 
 pkgver() {
-    git ls-remote "${url}" master | cut -f1 
+    git ls-remote "${url}" master | cut -f1
```
Comment (wizard-28) [inline packages/vega-strike-git/vega-strike-git.pacscript:9]: Why is the indentation 4 spaces?

File: packages/vega-strike-engine-git/vega-strike-engine-git.pacscript [hunk 3]
```diff
@@ -8,26 +8,26 @@ description="Vega Strike engine"
 maintainer="echometerain <echometer@disroot.org>"
 
 pkgver() {
-    git ls-remote "${url}" master | cut -f1 
+    git ls-remote "${url}" master | cut -f1
```
Comment (wizard-28) [inline packages/vega-strike-engine-git/vega-strike-engine-git.pacscript:11]: Why is the indentation 4 spaces?

File: packages/vega-strike-engine-git/vega-strike-engine-git.pacscript [hunk 4]
```diff
@@ -8,26 +8,26 @@ description="Vega Strike engine"
...
+  make
+  cd ..
 }
 
 install() {
-	cd build
-	sudo make install DESTDIR="$STOWDIR/$name"
-	sudo rm -rf "$STOWDIR/$name/usr/include"
-	sudo rm -rf "$STOWDIR/$name/usr/lib"
+  cd build || exit
```
Comment (wizard-28) [inline packages/vega-strike-engine-git/vega-strike-engine-git.pacscript:29]:
```suggestion
  cd build || return 1
```

**Review by wizard-28 — CHANGES_REQUESTED** [review]

File: packages/vega-strike-engine-git/vega-strike-engine-git.pacscript [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+  cmake --build "./build" -j"$(nproc)" -DCMAKE_BUILD_TYPE=Release
+  make
+  cd ..
+}
+
+install() {
+  cd build || return 1
+  sudo make install DESTDIR="${STOWDIR}/${name}"
+  sudo rm -rf "${STOWDIR}/${name}/usr/include"
+  sudo rm -rf "${STOWDIR}/${name}/usr/lib"
```
Comment (wizard-28) [inline packages/vega-strike-engine-git/vega-strike-engine-git.pacscript:32]: Umm... why?

Comment (echometerain) [reply packages/vega-strike-engine-git/vega-strike-engine-git.pacscript:32]: AUR had this

File: packages/vega-strike-git/vega-strike-git.pacscript [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+  git ls-remote "${url}" master | cut -f1
+}
+version="$(pkgver)"
+
+prepare() {
+  true
+}
+build() {
+  cmake "./CMakeLists.txt"
+  make
```
[the remark this answers is not in the document]
Comment (echometerain) [reply packages/vega-strike-git/vega-strike-git.pacscript:18]: the makefile here doesn't actually build anything, it just moves some files

File: packages/vega-strike-engine-git/vega-strike-engine-git.pacscript [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+  true
+}
+
+build() {
+  mkdir build
+  cd build || return 1
+  cmake "../engine"
+  cmake --build "./build" -j"$(nproc)" -DCMAKE_BUILD_TYPE=Release
+  make
+  cd ..
```
[the remark this answers is not in the document]
Comment (echometerain) [reply packages/vega-strike-engine-git/vega-strike-engine-git.pacscript:25]: image

File: packages/vega-strike-git/vega-strike-git.pacscript [hunk 8]
```diff
@@ -0,0 +1,23 @@
+name="vega-strike-git"
+url="https://github.com/vegastrike/Assets-Production.git"
+depends="make"
```
Comment (wizard-28) [inline packages/vega-strike-git/vega-strike-git.pacscript:3]: How does it depend on `make`

**Review by wizard-28 — CHANGES_REQUESTED** [review]

**Review by wizard-28 — APPROVED** [review]

**Pull request merged**